scania-digital-design-system/sdds
# Improvement/vars prefix sdds

**Describe pull-request**  
- CSS Vars name on colour, typography & logotypes now has -sdds prefix.

**How to test**
- Check css rendered in dist/styles/c-theme
- Go to bottom part find :root,html {} rules, see all vars has -sdds in front
- Check storybook and see if styling applied on all examples
